tests/backup_test.php

```diff
@@ -0,0 +1,236 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+namespace qtype_crossword;
+
+use qtype_crossword;
+
+defined('MOODLE_INTERNAL') || die();
+global $CFG;
+require_once($CFG->libdir . "/phpunit/classes/restore_date_testcase.php");
+require_once($CFG->dirroot . '/question/type/crossword/questiontype.php');
+
+/**
+ * Unit tests for backup/restore process in crossword qtype.
+ *
+ * @package qtype_crossword
+ * @copyright 2023 The Open University
+ * @license http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+class backup_test extends \restore_date_testcase {
+
+    /** @var qtype_crossword instance of the question type class to test. */
+    protected $qtype;
```

Comment:
This is only used in one place. Why bother with setUp and tearDown? Why not just do new qtype_crossword(); in the one place you need it? (Oh, and that tear-down is just unnecessary, by the way.)